Managers/NetworkServer.h

```diff
@@ -225,11 +223,12 @@ namespace RTE {
 
 		bool m_UseHighCompression; //!< Whether to use higher compression methods (default).
 		bool m_UseFastCompression; //!< Whether to use faster compression methods and conserve CPU.
+		bool m_UseDeltaCompression; //!<
```

Comment:
Even if you didn't get a weegee comment explanation for this, I think you can fill it in with similar to the settingsman one, right?